### PR TITLE
Feedback from forked repositories

### PR DESCRIPTION
**Pull request by cxxxr**
Hello.

First of all, I would like to express our sincere appreciation for creating this wonderful library.

We have forked this library and made some modifications to suit our needs. This PR is intended to provide feedback on the changes we made in our forked version to the original repository.

If there are any changes or concerns that should avoid, please let us know.

Thank you for considering our contribution. I look forward to your review.

Best regards.

### CODE REVIEW

**Comment by mmontone** [comment]
All in all changes look good to me.

This improves the stacktrace details, among other things, right?

I would make compression toggleable, adding an instance variable to sentry-client class, initialized to T perhaps. What do you think?

**Comment by cxxxr** [comment]
Thank you for the review.
I have added a compression option.
Please let me know if there is anything else I can do for you :pray:

**Comment by mmontone** [comment]
Merged. Thanks for your contribution!

**Comment by cxxxr** [comment]
Thank you for very much!

File: sentry-client.asd [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
+  :depends-on (:dexador
                :cl-json
                :cl-ppcre
                :uuid
                :local-time
-               :trivial-backtrace)
+               :trivial-backtrace
+               :babel
+               :salza2
+               :swank)
```
Comment (tdrhq) [inline sentry-client.asd:18]: Adding swank breaks images that are also using sly/slynk  :/

```
nhandled SB-KERNEL:SIMPLE-PACKAGE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                      {1000578083}>:
  "SWANK" is a package name, so it cannot be a nickname for "SLYNK".

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {1000578083}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK #<SB-KERNEL:SIMPLE-PACKAGE-ERROR "~S is a package name, so it cannot be a nickname for ~S." {100EA2F303}> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK SB-EXT:*INVOKE-DEBUGGER-HOOK* #<SB-KERNEL:SIMPLE-PACKAGE-ERROR "~S is a package name, so it cannot be a nickname for ~S." {100EA2F303}>)
2: (INVOKE-DEBUGGER #<SB-KERNEL:SIMPLE-PACKAGE-ERROR "~S is a package name, so it cannot be a nickname for ~S." {100EA2F303}>)
3: (CERROR "Ignore this nickname." SB-KERNEL:SIMPLE-PACKAGE-ERROR :PACKAGE #<PACKAGE "SLYNK"> :FORMAT-CONTROL "~S is a package name, so it cannot be a nickname for ~S." :FORMAT-ARGUMENTS ("SWANK" "SLYNK"))
4: (SB-IMPL::%ENTER-NEW-NICKNAMES #<PACKAGE "SLYNK"> ("SWANK"))
5: (RENAME-PACKAGE #<PACKAGE "SLYNK"> "SLYNK" ("SWANK"))
6: (ENSURE-SLYNK-PACKAGE-NICKNAMES)
7: (SB-FASL::LOAD-FASL-GROUP #S(SB-FASL::FASL-INPUT :STREAM #<SB-SYS:FD-STREAM for "file /home/circleci/project/build/asdf-cache/sbcl-2.0.1.debian-linux-x64/build/quicklisp/2023-06-18/dists/quicklisp/software/sly-20230618-git/contrib/slynk-retro.fasl" {100EA20F03}> :TABLE #(119 #<PACKAGE "SB-IMPL"> SB-IMPL::%DEFPACKAGE #1="SLYNK-RETRO" #2="CL" #3="SLYNK" #4="SLYNK-API" (#2# #3# #4#) (#1#) #5="/home/circleci/project/build/quicklisp/2023-06-18/dists/quicklisp/software/sly-20230618-git/contrib/slynk-retro.lisp" #<SB-KERNEL:LAYOUT for SB-C:DEFINITION-SOURCE-LOCATION {50301883}> #S(SB-C:DEFINITION-SOURCE-LOCATION :NAMESTRING #5# :INDICES 32769) ...) :STACK #(0 ENSURE-SLYNK-PACKAGE-NICKNAMES *SLYNK-REQUIRE-HOOK* (#6=#<FUNCTION ENSURE-SLYNK-PACKAGE-NICKNAMES>) #6# NIL *SLYNK-REQUIRE-HOOK* :IF-DOES-NOT-EXIST 47168 MERGE-PATHNAMES 46537 USER-HOMEDIR-PATHNAME ...) :NAME-BUFFER #("" "SYMBOL-VALUERE-HOOK*O-SLYNK*ES") :DEPRECATED-STUFF NIL :SKIP-UNTIL NIL) NIL)
8: (SB-FASL::LOAD-AS-FASL #<SB-SYS:FD-STREAM for "file /home/circleci/project/build/asdf-cache/sbcl-2.0.1.debian-linux-x64/build/quicklisp/2023-06-18/dists/quicklisp/software/sly-20230618-git/contrib/slynk-retro.fasl" {100EA20F03}> NIL NIL)
9: ((FLET SB-FASL::THUNK :IN LOAD))
10: (SB-FASL::CALL-WITH-LOAD-BINDINGS #<CLOSURE (FLET SB-FASL::THUNK :IN LOAD) {7F50648FD77B}> #<SB-SYS:FD-STREAM for "file /home/circleci/project/build/asdf-cache/sbcl-2.0.1.debian-linux-x64/build/quicklisp/2023-06-18/dists/quicklisp/software/sly-20230618-git/contrib/slynk-retro.fasl" {100EA20F03}>)
11: ((FLET SB-FASL::LOAD-STREAM :IN LOAD) #<SB-SYS:FD-STREAM for "file /home/circleci/project/build/asdf-cache/sbcl-2.0.1.debian-linux-x64/build/quicklisp/2023-06-18/dists/quicklisp/software/sly-20230618-git/contrib/slynk-retro.fasl" {100EA20F03}> T)
12: (LOAD #P"/home/circleci/project/build/asdf-cache/sbcl-2.0.1.debian-linux-x64/build/quicklisp/2023-06-18/dists/quicklisp/software/sly-20230618-git/contrib/slynk-retro.fasl" :VERBOSE NIL :PRINT NIL :IF-DOES-NOT-EXIST T :EXTERNAL-FORMAT :DEFAULT)
13: (UIOP/UTILITY:CALL-WITH-MUFFLED-CONDITIONS #<CLOSURE (LAMBDA NIL :IN UIOP/LISP-BUILD:LOAD*) {100EA1F62B}> ("Overwriting already existing readtable ~S." #(#:FINALIZERS-OFF-WARNING :ASDF-FINALIZERS)))
14: ((SB-PCL::EMF ASDF/ACTION:PERFORM) #<unused argument> #<unused argument> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "slynk/retro" "../contrib/slynk-retro">)
15: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
16: ((:METHOD ASDF/ACTION:PERFORM :AROUND (ASDF/LISP-ACTION:LOAD-OP ASDF/LISP-ACTION:CL-SOURCE-FILE)) #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "slynk/retro" "../contrib/slynk-retro">) [fast-method]
17: ((:METHOD ASDF/ACTION:PERFORM-WITH-RESTARTS (ASDF/LISP-ACTION:LOAD-OP ASDF/LISP-ACTION:CL-SOURCE-FILE)) #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "slynk/retro" "../contrib/slynk-retro">) [fast-method]
18: ((:METHOD ASDF/ACTION:PERFORM-WITH-RESTARTS :AROUND (T T)) #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "slynk/retro" "../contrib/slynk-retro">) [fast-method]
```

Comment (mmontone) [reply sentry-client.asd:18]: swank and slynk cannot coexist in same Lisp image?

**Pull request merged**